GoogleForCreators/web-stories-wp
# Redesign: Video Settings Panel

## Context



## Summary
Use checkbox from the design system for the loop setting.


## Relevant Technical Choices



## To-do




## User-facing changes
| Before | After |
|--------|-----|
| ![Screenshot 2021-03-11 at 17 04 22](https://user-images.githubusercontent.com/3294597/110807593-7cbcb280-8283-11eb-9226-c07db0f34d92.png) | ![Screenshot 2021-03-11 at 17 03 43](https://user-images.githubusercontent.com/3294597/110807622-85ad8400-8283-11eb-9e35-907fbf24c551.png) |


## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add a video
2. Verify the Video Settings panel looks as in [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3483%3A386)

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #6503
